examples/multisig-smart-account/webauthn-verifier/src/contract.rs

```diff
@@ -39,9 +45,13 @@ impl Verifier for WebauthnVerifierContract {
     ///
     /// * `true` if the signature is valid
     /// * `false` otherwise
-    fn verify(e: &Env, signature_payload: Bytes, key_data: BytesN<65>, sig_data: Bytes) -> bool {
+    fn verify(e: &Env, signature_payload: Bytes, key_data: Bytes, sig_data: Bytes) -> bool {
```

Comment:
I see that `verify` function is using `Self::KeyData` for the type of `key_data`. I think it is better use it like that also in here, because otherwise, providing the associated type seems pointless.